comps/comps-foreman-rhel7.xml

```diff
@@ -394,7 +394,6 @@
       <packagereq type="default">tfm-rubygem-trollop-doc</packagereq>
       <packagereq type="default">tfm-rubygem-unf-doc</packagereq>
       <packagereq type="default">tfm-rubygem-unf_ext-doc</packagereq>
-      <packagereq type="default">tfm-rubygem-unicode-doc</packagereq>
```

Comment:
looks unrelated?

Reply:
Yes, it was created automatically when running `comps_doc.sh` - the package doesn't exist so it may as well be removed from comps